test/integration/test_integration.py

```diff
@@ -226,3 +227,8 @@ def test_snapshots(vms):
         vm.revert_snapshot(snapshot_name)
         with snapshot(vm):
             pass
+
+
+def test_created_at(vms):
+    for vm in vms.values():
+        assert type(vm.created_at) == datetime.datetime
```

Comment:
`isinstance(vm.created_at, datetime.datetime)` would be better here. That allows for other datetime subclasses to be pulled in later on.

Reply:
I always forget about `isinstance` for some reason :man_shrugging: